### PR TITLE
Alt for images and a

### PR DESCRIPTION
**Pull request by dipanshubhola1009**

### CODE REVIEW

File: client/src/routes/Pages/FourOFour/FourOFour.js [hunk 1]
```diff
@@ -5,10 +5,10 @@ import './FourOFour.scss';
 
 const FourOFour = () => (
   <main class="four-o-four">
-    <img src={FourOFourVis} />
+    <img src={FourOFourVis} alt="Powered by DigitalOcean" />
     <h1>Oops, this connection just broke into numbers!</h1>
     <p>Lets get you back to Blaze to start sharing some files</p>
-    <a class="btn" href="/">Go to home</a>
+    <a class="btn" alt="DigitalOcean" href="/">Go to home</a>
```
Comment (blenderskool) [inline client/src/routes/Pages/FourOFour/FourOFour.js:11]: `alt` attributes are only for images. This change was unnecessary.

File: client/src/routes/Pages/components/Footer/Footer.js [hunk 2]
```diff
@@ -6,21 +6,21 @@ const Footer = () => (
   <footer>
     <div>
       <a href="https://m.do.co/c/ddb2a965377c" target="_blank" rel="noreferrer">
```
Comment (blenderskool) [inline client/src/routes/Pages/components/Footer/Footer.js:8]: As a matter of fact, the change was required here but leaving this all the other elements were changed :confused:

**Review by blenderskool — CHANGES_REQUESTED** [review]
@dipanshubhola1009 Thank you for PR but there are some erroneous changes here. Please try to understand the issue clearly to avoid extra changes.

The issue mainly mentioned the accessibility issues of **landing page**, and the suggestions I had left as comment applied to only the **specific elements marked in the Lighthouse report.**

In this PR, the changes have been made to all pages which is great but logically incorrect :confused:

File: client/src/routes/Pages/FourOFour/FourOFour.js [hunk 3]
```diff
@@ -5,10 +5,10 @@ import './FourOFour.scss';
 
 const FourOFour = () => (
   <main class="four-o-four">
-    <img src={FourOFourVis} />
+    <img src={FourOFourVis} alt="Powered by DigitalOcean" />
```
Comment (dipanshubhola1009) [inline client/src/routes/Pages/FourOFour/FourOFour.js:8]: It was required

**Review by dipanshubhola1009 — COMMENTED** [review]
Sorry for this. I will generate a new request.

File: client/src/routes/Pages/FourOFour/FourOFour.js (hunk 3 above)
Comment (blenderskool) [reply client/src/routes/Pages/FourOFour/FourOFour.js:8]: Ok, then "Powered by DigitalOcean" does not make sense for this image. It should be changed to "404 error image" I guess.

File: client/src/routes/Pages/Home/Home.js [hunk 4]
```diff
@@ -84,7 +84,7 @@ const Home = () => (
         <span>"</span>I built Blaze because I wanted a fast, radically different way to transfer files between my laptop and mobile<span>"</span>
 
         <footer>
-          <a href="https://github.com/blenderskool" target="_blank" rel="noopener noreferrer">
+          <a href="https://github.com/blenderskool" target="_blank"  rel="noopener noreferrer">
```
Comment (blenderskool) [inline client/src/routes/Pages/Home/Home.js:87]: This change is not required as it introduces double space between attributes.

File: client/src/routes/Pages/Home/Home.js [hunk 5]
```diff
@@ -99,7 +99,7 @@ const Home = () => (
         <h2 class="title">Let's get sharing!</h2>
         <h3 class="subtitle">No registrations required, completely free</h3>
 
-        <a href="/app" class="btn">
+        <a href="/app" alt="DigitalOcean" class="btn">
```
Comment (blenderskool) [inline client/src/routes/Pages/Home/Home.js:102]: Does not make sense here. Also, `alt` attributes don't work for anchor tags.

File: client/src/routes/Pages/components/Footer/Footer.js [hunk 6]
```diff
@@ -6,14 +6,14 @@ const Footer = () => (
...
-        <img class="digital-ocean" src="https://opensource.nyc3.cdn.digitaloceanspaces.com/attribution/assets/PoweredByDO/DO_Powered_by_Badge_white.svg" />
+        <img alt="Powered by DigitalOcean" class="digital-ocean" src="https://opensource.nyc3.cdn.digitaloceanspaces.com/attribution/assets/PoweredByDO/DO_Powered_by_Badge_white.svg" />
       </a>
     </div>
 
     <div>
       <span>
         © 2018 - 2020&nbsp;
-        <a href="https://github.com/blenderskool" target="_blank" rel="noreferrer">
+        <a  href="https://github.com/blenderskool" target="_blank" rel="noreferrer">
```
Comment (blenderskool) [inline client/src/routes/Pages/components/Footer/Footer.js:16]: This change is not required as it introduces double space between attributes.

File: client/src/routes/Pages/components/Header/Header.js [hunk 7]
```diff
@@ -19,7 +19,7 @@ function Header() {
 
   return (
     <header class="page-header">
-      <a class="brand" href="/">
+      <a class="brand"  href="/">
```
Comment (blenderskool) [inline client/src/routes/Pages/components/Header/Header.js:22]: This change is not required as it introduces double space between attributes.

**Review by blenderskool — CHANGES_REQUESTED** [review]
@dipanshubhola1009 I have marked some areas where the changes can be improved.

**Comment by blenderskool** [comment]
@dipanshubhola1009 any update here?

**Comment by blenderskool** [comment]
This PR is being closed as the issue was fixed by another PR.

**Pull request closed without merging**